src/naive/date.rs

```diff
@@ -1095,6 +1115,13 @@ impl NaiveDate {
     pub fn iter_weeks(&self) -> NaiveDateWeeksIterator {
         NaiveDateWeeksIterator { value: *self }
     }
+
+    /// Returns the [`NaiveWeek`] that the date belongs, starting with the
```

Comment:
"belongs to".

I think this could use some example to make to make the semantics clearer. Also should we rename the `weekday` argument to `start` or something here?